### PR TITLE
Fix for Issue #86 (per second quotas)

### PR DESCRIPTION
**Pull request by bgood**
This PR adds another branch to `getQuotas` to detect quotas that are tracked on a per second basis and the MQL to fetch them.

Fixes Issue #86

### CODE REVIEW

**Review by anuradha-bajpai-google — APPROVED** [review]

**Comment by vitalii-buchyn-exa** [comment]
Hello community,

Apologies, but when this is planned to be merged?

Thanks.

**Comment by bgood** [comment]
We are working on some extra validations of the fix, once they are finished we'll merge right away.

**Pull request merged**
